Src/Public/Start-LabHostConfiguration.ps1

```diff
@@ -57,6 +57,20 @@ function Start-LabHostConfiguration {
         foreach ($configuration in $labHostSetupConfiguation) {
 
             Import-LabDscResource -ModuleName $configuration.ModuleName -ResourceName $configuration.ResourceName -Prefix $configuration.Prefix -UseDefault:$configuration.UseDefault;
+            
+            ## Pending reboot switch being ignored. Fixed with this clause
+            if ( $IgnorePendingReboot ) {
+
+                if ( $configuration.Prefix -eq "PendingReboot" ) {
+                    WriteVerbose (
+                        "IgnorePendingReboot switch specified. Skipping DscResource: {0}" -f
```

Comment:
Can you add the localisation string to \Resources.psd1 file and update the reference using the `$localized` variable?